recipes/r-metaskat/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "0.80" %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: "r-metaskat"
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/MetaSKAT_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/MetaSKAT/MetaSKAT_{{ version }}.tar.gz
+  sha256: f8957d8e6586334360ca274110eee06d732aa333c410b99a3efb04da578e73c2
+
+build:
+  merge_build_host: true  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-skat
+  run:
+    - r-base
+    - r-skat
+    - {{ compiler('c') }}              # [not win]
```

Comment:
Actually, none of the run entries except the `r-` and `gcc-libs` lines should be needed; should have looked closer the first time.

Reply:
oh yeah, you are right, I now tested locally and pushing an update